### DISCUSSION TITLE
hyperlinks not working

### DISCUSSION THREAD

**Discussion by giladak** [kovidgoyal/kitty#5721] [Q&A]
hello, the Hyperlinks for simple files on kitty on my computer aren't working. did I forget to download something or is this possibly a bug?

**Comment by kovidgoyal** [comment]
You need to provide more information about your environment, what exactly you tried and what "not working" means.

**Comment by giladak** [comment]
I am working on Linux Ubuntu 20.04.4 . When I am hovering over a specific link, nothing happens. I downloaded kitty via the shell command that appears on the site.

![image](https://user-images.githubusercontent.com/116495877/204624553-74ba67ad-9b6f-4c9d-a6db-22eda099e499.png)

**Reply by tg0h** [reply]
https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda#apps

try providing the `--hyperlink=always` option to ls
